pkg/buildkit/buildkit_test.go

```diff
@@ -197,6 +197,18 @@ func TestNewClient(t *testing.T) {
 			cancel()
 			checkMissingCapsError(t, err, requiredCaps...)
 		})
+		t.Run("faulty addr", func(t *testing.T) {
+			t.Parallel()
+			addr := newMockBuildkitAPI(t) // this should return a faulty addr
```

Comment:
Can you have more details on this comment?
Its not clear from the call why this should be faulty and may produce something else in the future.

Reply:
That comment was left in by mistake, the address validation in `client.New` does not properly parse invalid addresses. If an address such as `300.300.300.300` is passed in, it shows no errors.

I've pushed a new commit that updates the test to be named `faulty key path` and added additional error validation for the tests.